### DISCUSSION TITLE
Bootstrap 5 - Popper.js v2.x

### DISCUSSION THREAD

**Discussion by nicolomonili** [twbs/bootstrap#31065] [General] [answered]
Hi, 
i have just finished taking a look at the first release of **Bootstrap 5**.
### A truly incredible update, thanks to all contributors and the community!

---

My question was about the Popper.js library, with which I'm having problems also in version 4 of Bootstrap (https://github.com/twbs/bootstrap/issues/29842 , https://github.com/twbs/bootstrap/issues/29313). Especially in the use of dropdowns with responsive tables.

The first release of Bootstrap 5 [still includes](https://v5.getbootstrap.com/docs/5.0/getting-started/introduction/#js) the version 1.x of Popper.js.

Since IE support has now been removed (and this seemed the biggest obstacle to be overcome in order to use version 2 of Popper.js), is it possible to think that compatibility with version 2 of Popper.js may be included in one of next releases of Bootstrap 5?

Thanks

**Comment by techboyg5** [comment]
Bootstrap 5 is in its first alpha right now. Many things will change from the first alpha to the 5.0 stable release.

**Comment by Johann-S** [comment, marked as answer]
Hi @nicolomonili 

We plan to upgrade to Popper.js v2 ASAP
